var/spack/repos/builtin/packages/enzyme/package.py

```diff
@@ -63,7 +64,10 @@ def libs(self):
 
     def setup_dependent_build_environment(self, env, dependent_spec):
         # Get the LLVMEnzyme and ClangEnzyme lib paths
-        llvm, clang = self.libs
+        if self.version >= Version("0.0.32"):  # TODO actual lower bound
+            llvm, clang, ldd = self.libs
+        else:
+            llvm, clang = self.libs
 
         if "LLVMEnzyme-" in clang:
```

Comment:
we may also need a swap like this for lld, right?

Reply:
That is a good point. There would be 6 permutations though. I wonder if there is a better way.

Reply:
My current proposed solution ([c48447c](https://github.com/spack/spack/pull/50041/commits/c48447c9c199bed8a22ee1e8520b825a94bde71c)) is to look for each library separately when setting the dependent build environment.